### ISSUE TITLE
[BUG] Oxy Trailer code has to be reset after every store

### ISSUE THREAD

**Issue by RoseiiNaytool** [ParadoxGamingRP/bugs-and-suggestions#2472]
### Your Discord Name

Roseii

### A clear and concise description of what the bug is

After every storm I have to reset the Oxy Trailer code so that anyone else can access the trailer. You can put in the same code as the one before but after city reset I have to reset it every time before the code will work for them again. There is no error in the F8 menu

### Steps to Reproduce the Bug

1. Put in code
2. Try to use code after city daily reset
3. Code doesn't work for anyone else
4. reset code so that everyone else can get into the trailer

### Expected Behavior

To be able to use the code to enter the trailer after daily trailer reset for other people

### Screenshots/Videos

No picture or video, no error in the F8 menu

**Comment by kevstax** [comment]
seems like the code doesn't even work through the same storm. I was able to get in the trailer after the code was reset but now the same code does not work anymore.

**Comment by RoseiiNaytool** [comment]
I haven't had an issue with it after I reset it after the storm, but that will help if different people are having different issues.

**Comment by tusktheshrew** [comment]
I had this issue when Colin tried to enter the code I submitted earlier.

**Comment by kevstax** [comment]
Yeah we’ve had to reset the pin every day since we got the trailer
